python/src/py_operators/mod.rs

```diff
@@ -26,30 +26,36 @@ fn construct_operator(
 ) -> Arc<PyObject> {
     Python::with_gil(|py| -> Arc<PyObject> {
         let locals = PyDict::new(py);
-        locals
+        if let Some(e) = locals
```

Comment:
Could you add a TODO that this function should return a Result? This isn't a good way to handle errors.